vmm/src/interrupt.rs

```diff
@@ -165,6 +205,26 @@ impl InterruptSourceGroup for MsiInterruptGroup {
         None
     }
 
+    #[cfg(target_arch = "x86_64")]
+    fn enable_resamplefd(
+        &self,
+        index: InterruptIndex,
+    ) -> vm_device::interrupt::Result<()> {
```

Comment:
Please address the clippy issue here with `cargo fmt`.

Reply:
Ok. I renamed the function name but forgot to run "cargo fmt".

Reply:
> Ok. I renamed the function name but forgot to run "cargo fmt".

@rbradford already done.